sdk/typescript/runtime/main.go

```diff
@@ -89,7 +92,7 @@ func (t *TypeScriptSdk) ModuleRuntime(ctx context.Context, modSource *ModuleSour
 	case Node:
 		return ctr.
 			// Install dependencies
-			WithExec([]string{"npm", "install"}).
+			WithExec([]string{"yarn"}).
```

Comment:
Could we use `yarn --production` here as well?

Reply:
> Could we use yarn --production here as well?

Unfortunately no we cannot, otherwise it fails with a weird node error that says a package is missing

Reply:
That's because the generated `package.json` doesn't look right:

```json
{
  "dependencies": {
    "typescript": "^5.3.2"
  },
  "devDependencies": {
    "@dagger.io/dagger": "./sdk"
  }
}
```

Is `@dagger.io/dagger` a dev dependency if it's *required* to actually run the module? IMO, it should be moved to a normal dependency, then we can switch this to `--production`.

Reply:
Yes but there will be no improvement in the performance by doing that, since we'll only have 2 main dependencies and no dev one